#!/usr/bin/env python

PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#        variable  type  level  description  default  min  max
gen.add("queue_size", int_t, 0, "queue size", 100, 1, 1000)
gen.add("threshold", double_t, 0, "Minimum likelihood", 0.5, 0.0, 1.0)
gen.add("negative", bool_t, 0, "Choose negative value", False)
exit(gen.generate(PACKAGE, PACKAGE, "PolygonArrayLikelihoodFilter"))
